docs/spark/spark-configuration.md

```diff
@@ -185,4 +185,5 @@ df.write
 | snapshot-property._custom-key_    | null            | Adds an entry with custom-key and corresponding value in the snapshot summary  |
 | fanout-enabled       | false        | Overrides this table's write.spark.fanout.enabled  |
 | check-ordering       | true        | Checks if input schema and table schema are same  |
-
+| isolation-level | null | Desired isolation level for Dataframe overwrite operations.  Null => no checks, `serializable` => check for concurrent inserts or deletes in destination partitions, `snapshot` => checks for concurrent deletes in destination partitions. |
```

Comment:
I think of the `null` case as useful for when the write is idempotent. For example, if you're consuming changed data from an upstream table hourly and writing a complete agg, then the change is idempotent and it doesn't matter what else happens to the partition (for dynamic overwrite).

Maybe there is a good way to state that use case for the "no checks" case?

Reply:
Sure, ```Null => no checks (idempotent writes)```?

Reply:
Yes? Do you think that's clear? Maybe "writes should be idempotent" or "for idempotent writes"?

Reply:
Sounds good, let me open another small pr for this